aspnetcore/security/authentication/social/facebook-logins.md

```diff
@@ -4,32 +4,36 @@ author: rick-anderson
 description: Tutorial with code examples demonstrating the integration of Facebook account user authentication into an existing ASP.NET Core app.
 ms.author: riande
 ms.custom: "seoapril2019, mvc, seodec18"
-ms.date: 03/04/2019
+ms.date: 12/02/2019
+monikerRange: '>= aspnetcore-3.0'
 uid: security/authentication/facebook-logins
 ---
+
 # Facebook external login setup in ASP.NET Core
 
 By [Valeriy Novytskyy](https://github.com/01binary) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-This tutorial with code examples shows how to enable your users to sign in with their Facebook account using a sample ASP.NET Core 2.0 project created on the [previous page](xref:security/authentication/social/index). We start by creating a Facebook App ID by following the [official steps](https://developers.facebook.com).
+This tutorial with code examples shows how to enable your users to sign in with their Facebook account using a sample ASP.NET Core 3.0 project created on the [previous page](xref:security/authentication/social/index). We start by creating a Facebook App ID by following the [official steps](https://developers.facebook.com).
 
 ## Create the app in Facebook
 
-* Navigate to the [Facebook Developers app](https://developers.facebook.com/apps/) page and sign in. If you don't already have a Facebook account, use the **Sign up for Facebook** link on the login page to create one.
+* Install [Microsoft.AspNetCore.Authentication.Facebook](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Facebook).
+
+* Navigate to the [Facebook Developers app](https://developers.facebook.com/apps/) page and sign in. If you don't already have a Facebook account, use the **Sign up for Facebook** link on the login page to create one.  Once you have a Facebook account, follow the instructions to register as a Facebook Developer.
 
-* Tap the **Add a New App** button in the upper right corner to create a new App ID.
+* From the **My Apps** menu select **Create App** to create a new App ID.
 
    ![Facebook for developers portal open in Microsoft Edge](index/_static/FBMyApps.png)
 
 * Fill out the form and tap the **Create App ID** button.
 
   ![Create a New App ID form](index/_static/FBNewAppId.png)
 
-* On the **Select a product** page, click **Set Up** on the **Facebook Login** card.
+* On your new App card, select **Add a Product**.  On the **Facebook Login** card, click **Set Up** 
```

Comment:
* On the new App card, select **Add a Product**.  On the **Facebook Login** card, click **Set Up**

Reply:
Avoid you, your expect when it adds clarity. Sometimes it sounds better with you/your. In this case, it's better without it.
If you agree with the suggestions batch add them from the files tab.